ntbs-service/wwwroot/source/Components/CascadingDropdown.ts

```diff
@@ -102,6 +103,17 @@ const CascadingDropdown = Vue.extend({
             values.forEach(item => optionInnerHtml += `<option value="${item.value}">${item.text}</option>`);
             return optionInnerHtml;
         },
+        setSelectToValueOrDefault: function (selectElement: HTMLSelectElement, targetValue: string) {
+            for (let i = 0; i < selectElement.options.length; i++) {
+                if (selectElement.options[i].value === targetValue) {
+                    selectElement.value = targetValue;
+                    return;
+                }
+            }

```

Comment:
I know this is code that you've copied from elsewhere, so possibly not worth updating, but something like this is cleaner using `find` or `findIndex`.

Reply:
The HTMLOptionsCollection doesn't have a find/findIndex method. The only method it has that works in IE is length (the others are add, remove, selectedIndex)

Reply:
Oh, I see... yeah, in that case this is about all you can do (unless you map to an array first)